spring-data-jdbc/src/main/java/org/springframework/data/jdbc/core/convert/BasicJdbcConverter.java

```diff
@@ -240,21 +251,6 @@ public Object readValue(@Nullable Object value, TypeInformation<?> type) {
 		return super.readValue(value, type);
 	}
 
-	/*
```

Comment:
I think we can even remove this API, because parent method also perform the same `null` check. Also if we assume that any external client also use this API, then they will not notice anything, because in this case they will just call parent `writeValue` directly.